ws-client/src/client.rs

```diff
@@ -373,38 +373,33 @@ fn process_response(
 
 	match manager.request_status(&response_id) {
 		RequestStatus::PendingMethodCall => {
-			let send_back_oneshot = manager.complete_pending_call(response_id).ok_or(Error::InvalidRequestId)?;
+			let send_back_oneshot = match manager.complete_pending_call(response_id) {
+				Some(Some(send)) => send,
+				Some(None) => return Ok(None),
+				None => return Err(Error::InvalidRequestId),
+			};
+
 			manager.reclaim_request_id(response_id);
 			let response = response.try_into().map_err(Error::Request);
-			match send_back_oneshot.map(|tx| tx.send(response)) {
```

Comment:
This was very stupid indeed by me, basically if it's fails it returns the message that was sent.

So fine to just ignore this.